compiler/luci/pass/src/ResolveFormerCustomOpPass.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveFormerCustomOpPass.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/AttrFusedActFunc.h>
+#include <luci/Profile/CircleNodeOrigin.h>
+
+#include <flatbuffers/flexbuffers.h>
+namespace
+{
+
+bool resolve_with_BroadcastTo(luci::CircleCustom *node)
+{
+  // check if input are support data types
+  auto input = loco::must_cast<luci::CircleNode *>(node->inputs(0));
+  if (input->dtype() != loco::DataType::FLOAT32)
+    return false;
+
+  // check if shape are support data types
+  auto shape = loco::must_cast<luci::CircleNode *>(node->inputs(1));
+  if (shape->dtype() != loco::DataType::S32 && shape->dtype() != loco::DataType::S64)
+    return false;
+
+  auto name = node->name();
+  assert(name.length() > 0);
+
+  auto broadcastTo = node->graph()->nodes()->create<luci::CircleBroadcastTo>();
+  broadcastTo->input(input);
+  broadcastTo->shape(shape);
+  broadcastTo->name(name);
+
```

Comment:
We should add an origin about this new op.
```cxx
luci::add_origin(broadcastTo, luci::get_origin(node));
```

Reply:
@mhs4670go 

Thank you a lot !! :)
Could you explain what function `luci::add_origin()` has?

Reply:
As I said in the offline, "origin" feature has been introduced for knowing where the newly created operator come from during the optimization process. 

`luci::add_origin` has two parameters - former is newly created one and the latter is the origin.